plugins/module_utils/redfish_utils.py

```diff
@@ -149,6 +151,41 @@ def patch_request(self, uri, pyld):
                     'msg': "Failed PATCH request to '%s': '%s'" % (uri, to_text(e))}
         return {'ret': True, 'resp': resp}
 
+    def put_request(self, uri, pyld):
+        req_headers = dict(PUT_HEADERS)
+        r = self.get_request(uri)
+        if r['ret']:
+            # Get etag from etag header or @odata.etag property
+            etag = r['headers'].get('etag')
+            if not etag:
+                etag = r['data'].get('@odata.etag')
+            if etag:
+                if self.strip_etag_quotes:
+                    etag = etag.strip('"')
+                req_headers['If-Match'] = etag
+        username, password, basic_auth = self._auth_params(req_headers)
+        try:
+            resp = open_url(uri, data=json.dumps(pyld),
+                            headers=req_headers, method="PUT",
+                            url_username=username, url_password=password,
+                            force_basic_auth=basic_auth, validate_certs=False,
```

Comment:
why harcoding `validate_certs=False`?

Reply:
This is as per already existing functions in redfish_utils.py. Refer post_request() is redfish_utils.py.